app/src/app/[lng]/cities/page.tsx

```diff
@@ -0,0 +1,228 @@
+"use client";
+import {
+  Box,
+  Button,
+  Heading,
+  Icon,
+  IconButton,
+  Link,
+  Table,
+  Text,
+} from "@chakra-ui/react";
+import React, { useMemo, useState } from "react";
+import { useTranslation } from "@/i18n/client";
+import {
+  useGetAllCitiesInSystemQuery,
+  useGetOrganizationsQuery,
+} from "@/services/api";
+import ProgressLoader from "@/components/ProgressLoader";
+import DataTable from "@/components/ui/data-table";
+import {
+  MenuContent,
+  MenuItem,
+  MenuRoot,
+  MenuTrigger,
+} from "@/components/ui/menu";
+import { MdMoreVert, MdOutlineGroup } from "react-icons/md";
+import MoveCityModal from "@/app/[lng]/cities/MoveCityModal";
+
+const CitiesPage = ({ params: { lng } }: { params: { lng: string } }) => {
+  const { t } = useTranslation(lng, "admin");
+  const { data, isLoading } = useGetAllCitiesInSystemQuery({});
+  const { data: organizationData, isLoading: isLoadingOrganizationData } =
+    useGetOrganizationsQuery({});
```

Comment:
You can leave out the empty object here I think
```suggestion
    useGetOrganizationsQuery();
```